packages/api/src/config/brevo-module.config.ts

```diff
@@ -15,6 +15,7 @@ export interface BrevoModuleConfig {
             };
             allowedRedirectUrl: string;
             redirectUrlForImport: string;
+            folderId: number;
```

Comment:
this is a breaking change. 
I would make it optional and set it to default 1 in the package, not in the demo

Reply:
288630f670ac01e9738d773c180115c2bdf41a48